src/modules/Dashboard/features/Applet/Schedule/Calendar/EventWrapper/EventWrapper.tsx

```diff
@@ -20,11 +20,43 @@ export const EventWrapper = ({
 
   const tooltipTitle = (
     <>
-      <Box>{`${format(start, DateFormats.Time)} - ${format(end, DateFormats.Time)}`}</Box>
+      <Box>{`${startTime ?? format(start, DateFormats.Time)} - ${
+        endTime ?? format(end, DateFormats.Time)
+      }`}</Box>
       <Box>{title}</Box>
     </>
   );
 
+  const renderEventWrapper = () => {
+    if (timeView && isHiddenInTimeView) {
+      return <Box ref={emptyRef} />;
+    }
+
+    return (
+      <Box
+        className="event-wrapper"
+        ref={childrenRef}
+        data-id={id}
+        data-start={start}
+        data-end={end}
+        data-testid={`dashboard-calendar-${id}`}
+      >
+        <Tooltip followCursor tooltipTitle={tooltipTitle} placement="top">
```

Comment:
could you please update type for `tooltipTitle` to React.ReactNode, it's more appropriate?